server/rest/src/test/java/org/infinispan/rest/RestOperationsTest.java

```diff
@@ -440,4 +440,49 @@ public void testConfigConverter() throws Exception {
       assertEquals(2, jsonNode.findValue("string-keyed-jdbc-store").size());
    }
 
+   @Test
+   public void testWeakCounter() throws Exception {
+      String url = String.format("http://localhost:%d/rest/v2/counters/weak", restServer().getPort());
+      ContentResponse response = client.newRequest(url)
```

Comment:
not sure if it makes much sense to test the "reset" at the beginning. we aren't sure if it works or not since there is no change in the counter's value.